ego/bundle/main.go

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Edgeless Systems GmbH.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//go:build linux
+// +build linux
+
+package main
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"debug/elf"
+	"io"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	"ego/cli"
```

Comment:
> I am not sure where to draw the line for code reuse. Many parts, especially for the unit tests are copied from the CLI or main EGo package to avoid calling `NewCLI`, though it is essentially the same code. On the other hand, I also was not sure whether to create an internal package to re-use functions, given that the bundle executable is supposed to pretty much stand on its own... I don't know, let me know in the review about your opinions.

I think we should not include the cli package here, but also not copy too much code. So I would move some code from cli to new package(s) for reuse.